<?xml version="1.0"?>
<launch>

    <node pkg="gps_common" type="utm_odometry_node" name="utm_odometry_node" output="screen" >
        <remap from='fix' to='/sensors/an_device/NavSatFix' />
        <remap from='odom' to='utm_odom' />
        <param name='frame_id' value='world' />
        <param name='child_frame_id' value='utm_odom' />
    </node>

</launch>
